agent_build_refactored/docker_image_builders.py

```diff
@@ -259,8 +259,8 @@ def build_container_filesystem(self, output_path: pl.Path):
             shutil.rmtree(package_root_path)
         package_root_path.mkdir(parents=True)
 
-        # Build 'LFS-structured' filesystem.
-        build_linux_lfs_agent_files(
+        # Build 'HFS-structured' filesystem.
```

Comment:
Looks like a typo here: HFS-structured => FHS-structured

By the way, FHS refers to https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard right?  What does LFS refer to?

Reply:
Oops, yes it does, fixed.  Thanks!